tools/make-edge.sh

```diff
@@ -21,8 +21,10 @@ cp platform/edge/*.js   $DES/js/
 cp -R platform/edge/img $DES/
 cp platform/edge/*.html $DES/
 cp platform/edge/*.json $DES/
+cp -R platform/edge/css     $DES/
```

Comment:
For consistency with the JS line could this be:
```
cp -R platform/edge/*.css $DES/css
```
And then move the `vapi-popup.css` file to the `edge` folder.